### ISSUE TITLE
Visual issue with Zelda Majora's Mask

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#443]
Originally reported on Google Code with ID 444

```
Describe your system:
 - Windows 7
 - 32-bit:
 - Mupen64Plus 0.5.1:
- Plugins used:

For visual artifacts, give:
 - ZELDA MAJORA'S MASK
 - 13FAB67E603B002CEAF0EEA84130E973
 - Maybe layers?

Describe the problem:

- When I use the "Lens of Truth", a new layer/mask or something like that should appear.
But nothing happens and this layer only appears when I die.

```

Reported by `alberto.msn` on 2011-07-12 14:05:38

**Comment by richard42** [comment]
```
Mupen64Plus 0.5.1? Are you sure that you are right here? Since you don't provid any
further information, I will just assume that this is a duplicate of 407
```

Reported by `sven@narfation.org` on 2011-07-12 20:44:27
- **Status changed**: `Duplicate`
- **Merged into**: #407

**Comment by richard42** [comment]
```
ok, you are right. Is the same issue, but in two different games.

I don't know what information I must provid. I just filled the 

Sorry about if I seem lost, is the first time I use this web.
```

Reported by `alberto.msn` on 2011-07-13 22:14:32
